source/tools/administration-interfaces.txt

```diff
@@ -414,16 +414,16 @@ Third-Party Commercial Tools
 
 - <http://3t.io/data-compare/>
 
-3T Data Compare is a multi-platform data comparison tool for MongoDB,
-and includes the following features:
+3T Data Compare is a MongoDB data comparison tool for ``Mac``, ``Windows``, and ``Linux``.
```

Comment:
The phrase 'is a MongoDB data comparison tool' is not acceptable as it implies that MongoDB is the source; please find alternate phrasing.